src/gp-model/main.py

```diff
@@ -201,7 +201,7 @@ def predict_model(m, tenant, airqloud, aq_id, poly, x1, x2, y1, y2):
     result.append(result_builder)
 
     db = connect_mongo(tenant)
-    collection = db["gp_predictions"]
+    collection = db["gp_model_predictions"]
```

Comment:
Hi @uman95 , the old collection has some old "dirty" data and I don't have access to delete it, so Martin suggested I just create a new one